assets/dev/js/frontend/handlers/base-swiper.js

```diff
@@ -0,0 +1,61 @@
+import BaseHandler from './base';
```

Comment:
If there is common settings/elements, consider moving them here.

Reply:
There aren't any. If we make a decision to change control names to unify all of the Swiper widgets, we would probably be able to move a lot more to the base. Unfortunately, this is currently not possible